src/components/Header/BaseHeader.jsx

```diff
@@ -57,6 +64,7 @@ const BaseHeader = ({
                 <a
                   className="rvt-lockup"
                   href={homeUrl}
+                  onClick={homeUrlOnClick}
```

Comment:
We only want to prevent default if the `onClick` is set.

```suggestion
                  onClick={onClick && homeUrlOnClick}
```

Reply:
`homeUrlOnClick` will be `null` if `onClick` is falsy. See the code where it is set up.

Reply:
Ah, I see that now, thank you.